app/model/application.js

```diff
@@ -79,12 +87,16 @@ ApplicationSchema.statics.fetchReleases = function(application) {
 }
 
 ApplicationSchema.statics.parseAppHubFileIfPossible = function(application) {
-  return Promise.try(() => {
-    // Parse the .desktop file
-    const appHubFileBuf = new Buffer(application.appHubFileResult.content, 'base64');
-    const appHubData = JSON.parse(appHubFileBuf.toString());
-    application.priceUSD = appHubData.priceUSD;
-    delete application.appHubFileResult;
+  const fullName = application.github.fullName;
+  return gh.request(`GET /repos/${fullName}/contents/.apphub`, {
+    token: application.github.APItoken,
+  })
+  .then(appHubFile => {
+    // Parse the .apphub file
+    const desktopFileBuf = new Buffer(appHubFile.content, 'base64');
+    const appHubFileJSON = JSON.parse(desktopFileBuf.toString())
+    application.priceUSD = appHubFileJSON.priceUSD
+    application.label = appHubFileJSON.issueLabel;
     return application;
   })
   .catch(() => application);
```

Comment:
since you got rid of the .try(), this catch is not needed. in fact, you definitely don't want it here, because if the gh.request promise fails, you want to fail loudly — this function should only be called on repositories that are known to have .apphub files. (might be good to remove the `ifPossible` from the function name to reflect this?)